draft-ietf-rats-uccs.md

```diff
@@ -81,10 +83,10 @@ informative:
 
 --- abstract
 
-CBOR Web Token (CWT, RFC 8392) Claims Sets sometimes do not need the
-protection afforded by wrapping them into COSE, as is required for a true
-CWT.  This specification defines a CBOR tag for such unprotected CWT
-Claims Sets (UCCS) and discusses conditions for its proper use.
+CBOR Web Token (CWT, RFC 8392) and JSON Web Token (JWT, RFC7519) Claims Sets sometimes do not need the
+protection afforded by wrapping them into COSE or JOSE, as is required for a true
+CWT or JWT.  This specification defines a CBOR and JSON for such unprotected
```

Comment:
_"a CBOR and JSON"_ seems too vague.

I guess there's more than one thing that is defined here:
* a CBOR tag for UCCS
* the CDDL for the claims-set
* the glue CDDL to plug UCS into EAT

Reply:
Normally we expect RFCs to have implementations behind them. Are there implementations for UCCS and UJCS?

Reply:
ctoken/xclaim implements UCCS. It has a very basic implementation of UJCS encoding, but no implementation of UJCS decoding.